PR capstone 1 conference

ivanmvh opened the pull request:
In this capstone project  I am going to build  an online website for a conference. Microverse provide some design guidelines to create the website, but I must personalize the content, i.e., instead of a conference I can build a website for a concert or for a web development course. It's very important the I personalize my project so that I have something unique in my portfolio to share with potential employers during job searching.

I should personalize the content of my page. Choose a topic that is different than the one in the original design.

I should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

Colors.
Typography: font face, size and weight (Suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
Layout: composition and space between elements.
The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

I can use a CSS framework (for example Bootstrap) for styling, if I want to.

I should build only these 2 pages:

The home page.
The about page.
Each of these pages should have versions for 2 different screen sizes:

Mobile: up to 768px wide.
Desktop: 768px or wider.
Interactions

Links
The home page should have a link in the menu to the about page.
The logo in the header links to the home page.
Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
There are no guidelines for the mobile menu in the docs, but I should implement it so it is consistent with the design (colors, typography, spacings, etc.).
The mobile menu should have a close (X) button that closes the menu.
Dynamic page

The section "Featured speakers" should be created dynamically in JavaScript.
I should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

ivanmvh committed:
Build directory structure

ivanmvh committed:
Create images for the project

ivanmvh committed:
Build structure of sass files

ivanmvh committed:
Create Top Seccion - Tag for Mobile menu - Nav Container

ivanmvh committed:
Build top bar - menu - header

ivanmvh committed:
Create nav-var , header for desktop version

ivanmvh committed:
Build  program section fo mobile version

ivanmvh committed:
Building Program section fror mobile and desktop version

ivanmvh committed:
Create _sponsors.scss

ivanmvh committed:
Adjust to sponsor section anf footer

ivanmvh committed:
Build dinamicacly dancers section wit js

ivanmvh committed:
Ajust styles for dancers section - mobile menu

ivanmvh committed:
Buid about sestion

ivanmvh committed:
Adjust styles of about section

ivanmvh committed:
Ajust about  section

ivanmvh committed:
Fixing details - images

ivanmvh committed:
Fixing some details

ivanmvh committed:
Fixing linters errors

mckent05 commented on src/js/main.js:
```diff
@@ -0,0 +1,123 @@
+              <div class="upper-part">
+                <div class="dancer-image-container">
+                  <div class="dancer-bg"></div>
+                  <img class="dancer-image" src="${array[i].image}" alt="${array[i].name} image" height="225" width="225">
```
-For improved SEO principles, It's not advisable to use inline styling in our HTML code. I'd suggest you keep all styling in your CSS file. Please fix this  👍 👍

ivanmvh replied:
ok thank you

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+
+  </head>
+  <body>
+    <div class="mobile-menu">
+      <i class="fa-solid fa-xmark"></i>
+    </div>
+    <div class="top-section desktop">
+      <img class="top-icon" src="./src/img/icons/fb-icon.svg" alt="facebook icon" height="10px" width="10px">
+      <img class="top-icon" src="./src/img/icons/twitter-icon.svg" alt="twitter icon" height="10px" width="10px">
+      <ul>
+        <li>English</li>
+        <li>My page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
```
- To improve your HTML structure and for better accessibility, It's advisable to put all these elements inside a `nav` tag. Please try to fix this 👍 👍

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+          <li><a href="#">News</a></li>
+        </ul>
+      </div>
+    </nav>
```
- For better accessibility and to improve the SEO of your site, I'd suggest that you wrap your `nav` tag in a `header` tag, please try to fix this 👍 👍 
- Your navbar does not exactly look like what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your Project 👇:
-- The topbar with social links has more margins on the right. Please fix this 🙏 
-- You are missing one last link for the bottom navbar with the nav links. Please fix this 😄 
-- Please ensure that your links are arranged the way they are in the design template 👍 👍 
<img width="949" alt="tango nav" src="https://user-images.githubusercontent.com/73607512/197237364-4b35503a-7780-4696-b4c7-2273562f7faa.PNG">

Design Template:
<img width="628" alt="navbar behance" src="https://user-images.githubusercontent.com/73607512/197237461-dd040556-01e4-41d7-b359-77d8d6183d0d.PNG">

- The nav for the mobile version also doesn't match the given design template, please try to fix this 😄 

Your project:
-- Please remove the logo from the mobile version nav as seen in your project
<img width="219" alt="tango mobile nav" src="https://user-images.githubusercontent.com/73607512/197238249-34b9ad93-79eb-4135-9ede-ceccc576de0d.PNG">

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+        </ul>
+      </div>
+    </nav>
+    <section id="header">
+      <div>
+        <p class="blockquote">"Put on your dancing shoes!!!"</p>
+        <h1 class="header-class">International <br> <span>Tango</span> <br> Festival</h1>
+      </div>
+      <div class="introduction">
+        <p><span class="quote">"The most important Tango event in South America"</span></p>
+        <p>The Internatiuonal Tango Festival will be held in Bogota, Colombia. The most famous milongueros from Colombia will present in this magnificent event.</p>
+      </div>
+      <div class="place">
+        <p class="date">2022.11.01 ~ 06 (monday to sunday)</p>
+        <p class="salon">@ Hotel Tequendama, Bogota , Colombia</p>
+      </div>
+    </section>
```
- This section of your website doesn't match the design in the given template. While it is good to personalize the contents of your site, please remember that you are required to stick to the general design layout (background image, text color, font size, font weight, margins, borders e.t.c) as seen in the design template. Also, remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

# Your project(Please fix as it may apply for the about page) 👇:
-- There is a background image in the design template that is missing in your project 👍 
--Please stick to the correct font-color, font-size and font-weight for the texts in this section 🙏 
-- The bottom part of this section is overflowing into the next section. Please fix this 👍 
<img width="944" alt="tango home header" src="https://user-images.githubusercontent.com/73607512/197239242-171eb668-9e5d-4c24-9430-1244ad02f10f.PNG">

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Milonga</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+          <a class="program-link desktop" href="#">See the whole program</a>
+          <button class="program-button">
+            <a class="program-link mobile" href="#">Join International Tango Festival</a>
+          </button>
+        </div>
+        <!-- cross bg -->
+      </section>
+      <!-- Program Section -->
+
+      <!-- dancers section -->
+      <section id="dancers">
+        <h2 class="dancers-heading">Guest dancers</h2>
+        <div class="line"></div>
+        <div class="dancers"> </div>
+      </section>
+      <!-- dancers section -->
+
+      <!-- sponsors section --> 
+      <section id="sponsors">
+        <h3 class="sponsors-heading">Sponsors</h3>
+        <div class="line"></div>
+        <div class="sponsors-logos">
+          <img class="google-logo" src="./src/img/sponsors/sponsor-5.png" alt="Google logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-1.png" alt="Airbnb logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-2.png" alt="GitHub logo" width="250" height="250">
+          <img src="./src/img/sponsors/sponsor-3.png" alt="Firefox logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-4.png" alt="Micorvese Logo" width="200" height="150">
+        </div>
+      </section>
+      <!-- sponsors section -->
+    </main>
```
- For improved HTML structure, I'd advise that you include the first section in the `main` tag since you are making use of it, It is also a main part of your HTML structure 🙏

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+
+          <!-- programs-container -->
+          <div class="programs-container">
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_01.png" alt="lecture icon">
+              <h3 class="program-card-heading">Workshop</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_02.png" alt="exhibition icon">
+              <h3 class="program-card-heading">Milonga Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_03.png" alt="forum icon">
+              <h3 class="program-card-heading">Tango Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_04.png" alt="workshop icon">
+              <h3 class="program-card-heading">Vals Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Milonga</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
```
For semantic reasons and to improve your site's accessibility, I'd suggest that you use an `article` tag in place of a `div` tag for each of the program card to avoid running into a div soup. Please fix this 👏

ivanmvh replied:
ok

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+      </section>
+      <!-- Program Section -->
+
+      <!-- dancers section -->
+      <section id="dancers">
+        <h2 class="dancers-heading">Guest dancers</h2>
+        <div class="line"></div>
+        <div class="dancers"> </div>
+      </section>
```
- The dancer's section of your project doesn't look exactly like the one in the design template.  This section should have 2 columns in a row of 3. You have aligned them in a single column in a row of 6. Please fix this issue 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

# Your project 👇:
-- Also for the mobile version, the 'description' text should be beside the dancer/speaker image, you have aligned it at the bottom of the image. Please try to fix this issue 👍 👍 
(Desktop version)    

<img width="948" alt="tango speakers" src="https://user-images.githubusercontent.com/73607512/197241011-2b835de5-7b7b-4ad9-afdf-371a86894e61.PNG">


(Mobile version) 

<img width="221" alt="tango speaker mobile" src="https://user-images.githubusercontent.com/73607512/197241449-b1461119-4a7e-46d2-a90c-ec5b5c45b848.PNG">


 # Design Template:

(Desktop version):

<img width="539" alt="speakers behance" src="https://user-images.githubusercontent.com/73607512/197242395-342f07cb-16a1-4075-87b5-5f97a574124a.PNG">


(Mobile version):
<img width="281" alt="speakers behance mobile" src="https://user-images.githubusercontent.com/73607512/197242594-6597558a-144b-4480-bef0-69c05e8602a3.PNG">

ivanmvh replied:
ok

mckent05 commented on index.html:
```diff
@@ -0,0 +1,152 @@
+      <section id="sponsors">
+        <h3 class="sponsors-heading">Sponsors</h3>
+        <div class="line"></div>
+        <div class="sponsors-logos">
+          <img class="google-logo" src="./src/img/sponsors/sponsor-5.png" alt="Google logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-1.png" alt="Airbnb logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-2.png" alt="GitHub logo" width="250" height="250">
+          <img src="./src/img/sponsors/sponsor-3.png" alt="Firefox logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-4.png" alt="Micorvese Logo" width="200" height="150">
+        </div>
```
- To improve your site's accessibility, I'd advise you wrap these images, each in a `li` tag, and all `li` tags be wrapped in one `ul` tag. Please look into this 👍 👍 
- I noticed some of your images have different sizes from the others, please ensure that all images are of the same size so the section can look much more like the one in the design template 😄 

Your Project:
<img width="945" alt="tango home sponsor" src="https://user-images.githubusercontent.com/73607512/197243239-5fe7563d-2f30-413b-8a04-17d710a29447.PNG">

- F[ollowing HTML best practices](https://worldofdev.info/6-reasons-why-you-shouldnt-style-inline/#:~:text=Inline%20styles%20increase%20the%20size,include%20them%20in%20every%20paragraph.), I suggest you do not apply styling in your HTML code, please keep all styling in your CSS file. Please, try to fix this 👍 👍

ivanmvh replied:
ok

mckent05 commented on about.html:
```diff
@@ -0,0 +1,139 @@
+        </div>
+      </section>
+      <!-- sponsors section -->
+      <section class="about-sponsors" id="sponsors">
+        <h3 class="sponsors-heading">Sponsors</h3>
+        <div class="line"></div>
+        <div class="sponsors-logos">
+          <img class="google-logo" src="./src/img/sponsors/sponsor-1.png" alt="Google logo" width="200" height="200">
+          <img src="./src/img/sponsors/sponsor-2.png" alt="Airbnb logo" width="200" height="100">
+          <img src="./src/img/sponsors/sponsor-3.png" alt="GitHub logo" width="100" height="100">
+          <img src="./src/img/sponsors/sponsor-4.png" alt="Firefox logo" width="100" height="100">
+          <img src="./src/img/sponsors/sponsor-5.png" alt="Micorvese Logo" width="400" height="100">
+        </div>
+      </section>
```
- For the about page, this section isn't included in the design template. Kindly remove this section 🙏 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Design Template:
<img width="554" alt="about behance footer" src="https://user-images.githubusercontent.com/73607512/197244798-b5208603-dab0-4cfd-a24f-21b15ab6fb2e.PNG">

ivanmvh replied:
ok

mckent05 reviewed (changes requested):
Hi @ivanmvh,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost there](https://user-images.githubusercontent.com/73607512/197245107-134ea484-a70b-4376-8f46-c437952dcc89.gif)

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Descriptive PR title and summary 👍 👍 


## Required Changes ♻️
- According to the project requirements, please deploy the live version of your project on GitHub pages and include the link in your README file 👍 👍 
- The project requirements also require that you make a loom video of no longer than 5mins, explaining your project and highlighting a piece of code you are proud of. Please include a link to the video in your README or in the PR summary. 🙏 
- To make your README look more professional, please include steps on how your project can be cloned locally on a computer 😄 😄 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ivanmvh committed:
Do changes suggested by the reviewer

# Color accord design
## change background image
# no show logo on mobile version
- -For improved SEO principles, It's not advisable to use inline styling in our HTML code. I'd suggest you keep all styling in your CSS file. Please fix this 👍 👍

ivanmvh committed:
Fixing linters errors

ivanmvh committed:
Fixing linters errors

ivanmvh committed:
fixing linters errors

ivanmvh committed:
Fixing linters errors

ivanmvh committed:
Add campaing button to menu

ivanmvh committed:
Fixing campaign button

ivanmvh committed:
Fixing linter errors

Fixing linter errors

emmiiorji reviewed (approved):
Hi @ivanmvh, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 
![congratulations](https://user-images.githubusercontent.com/20554583/197318511-6afdd6f2-aa6f-4da6-8061-4899dec8c651.gif)


## Good Highlights✔

- Linter checks are passing
- Good commit message
- Adherence to GitHub flow
- Required changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
